migrations/1603438527057-add-browser-os-info.js

```diff
@@ -0,0 +1,15 @@
+const db = require('./db')
+
+exports.up = function (next) {
+  var sql = [
+    'ALTER TABLE user_tokens ADD COLUMN browser_version text',
+    'ALTER TABLE user_tokens ADD COLUMN os_version text',
```

Comment:
We probably want to save the user_agent as a whole and do the parsing on the front-end project for display only purposes.